examples/imagenet/train.py

```diff
@@ -73,8 +76,9 @@ def init(*args):
 
 
 def cross_entropy_loss(logits, labels):
-  return -jnp.sum(
-      common_utils.onehot(labels, num_classes=1000) * logits) / labels.size
+  xentropy = optax.softmax_cross_entropy(
+      logits, common_utils.onehot(labels, num_classes=NUM_CLASSES))
```

Comment:
Could you specify these by keyword?
(It's all too easy inverting them, like thinking of `H(P, Q)` with `P~labels` and `Q~softmax(logits)`)

Reply:
Done